src/main/java/com/saiko/bidmarket/common/util/CookieUtils.java

```diff
@@ -0,0 +1,65 @@
+package com.saiko.bidmarket.common.util;
+
+import java.io.Serializable;
+import java.util.Base64;
+import java.util.Optional;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import org.apache.commons.lang3.SerializationUtils;
+
+public class CookieUtils {
+
+  public static Optional<Cookie> getCookie(HttpServletRequest request, String name) {
+    Cookie[] cookies = request.getCookies();
+
+    if (cookies != null && cookies.length > 0) {
+      for (Cookie cookie : cookies) {
+        if (cookie.getName().equals(name)) {
+          return Optional.of(cookie);
+        }
+      }
```

Comment:
```suggestion
      return Arrays.stream(cookies)
            .filter(cookie -> cookie.getName().equals(name))
            .findFirst();
```
스트림쓰면 더 가독성있게 쓸 수 있을것 같습니다